iina/PrefKeyBindingViewController.swift

```diff
@@ -301,6 +301,17 @@ class PrefKeyBindingViewController: NSViewController, MASPreferencesViewControll
       }
       // split
       let splitted = line.characters.split(separator: " ", maxSplits: 1)
+      if splitted.count < 2 {
+        let alert = String(format: NSLocalizedString("alert.keybinding_config_error", comment: "Error setting keybinding configuration"), currentConfName)
+        Utility.showAlert(message: alert)
```

Comment:
Seems that this `showAlert` will also crash...

I'll add a workaround and merge.